dataprocess/dataprocess.go

```diff
@@ -0,0 +1,38 @@
+// Package dataprocess manages code related to the data-accepting process
+package dataprocess
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"os"
+
+	pc "go.viam.com/rdk/pointcloud"
+)
+
+// WritePCDToFile Encodes the pointcloud and then saves it to the passed filename.
```

Comment:
[nit] Reflect the style in RDK

```suggestion
// WritePCDToFile encodes the pointcloud and then saves it to the passed filename.
```